packages/amplify-util-uibuilder/src/commands/utils/codegenResources.ts

```diff
@@ -26,9 +26,9 @@ import {
   ReactThemeStudioTemplateRendererOptions,
 } from '@aws-amplify/codegen-ui-react';
 import { printer } from '@aws-amplify/amplify-prompts';
-import { $TSContext, AmplifyCategories, AmplifySupportedService, stateManager } from 'amplify-cli-core';
+import { $TSContext } from 'amplify-cli-core';
 import { getUiBuilderComponentsPath } from './getUiBuilderComponentsPath';
-import { AmplifyStudioClient } from '../../clients';
+import { ModelIntrospectionSchema } from '@aws-amplify/appsync-modelgen-plugin';
```

Comment:
is declaring `@aws-amplify/appsync-modelgen-plugin` as dev dependency correct if you use it here ?

Reply:
It was added as a dev dependency because only a type is imported. Do you suggest another approach?

Reply:
gotcha. resolving this then.